### DISCUSSION TITLE
Version 0.7.0

### DISCUSSION THREAD

**Discussion by stefanhendriks** [stefanhendriks/Dune-II---The-Maker#535] [General]
This version improves gfx, introduces option menu, rewrites controls, introduces event system and compiles under Linux. 
[See the full changelog for all 70+ changes!](https://github.com/stefanhendriks/Dune-II---The-Maker/wiki/v0.7.0)

<hr /><em>This discussion was created from the release <a href='https://github.com/stefanhendriks/Dune-II---The-Maker/releases/tag/v0.7.0'>Version 0.7.0</a>.</em>

**Comment by dune2alex** [comment]
Great Job !

**Reply by stefanhendriks** [reply]
thanks!
